src/Wellcome.Dds/IIIFServerSmokeTest/SmokeTester.cs

```diff
@@ -0,0 +1,465 @@
+using System.Diagnostics;
+using System.Net;
+using System.Xml.Linq;
+using IIIF.Presentation.V3;
+using IIIF.Serialisation;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json.Linq;
+using Wellcome.Dds.Common;
+using Wellcome.Dds.IIIFBuilding;
+
+namespace IIIFServerSmokeTest;
+
+public class SmokeTester
+{
+    private UriPatterns uriPatterns;
+
+    public SmokeTester(string publicIIIFRoot, string wellcomeCollectionApi, string apiWorkTemplate)
+    {
+        var options = Options.Create(new DdsOptions
+        {
+            LinkedDataDomain = publicIIIFRoot,
+            WellcomeCollectionApi = wellcomeCollectionApi,
+            ApiWorkTemplate = apiWorkTemplate
+        });
+        uriPatterns = new UriPatterns(options);
+    }
+
+    public async Task<Result> Test(WorkFixture fixture)
+    {
+        var result = new Result();
+        if (fixture.Skip)
+        {
+            result.Add($"SKIPPING fixture {fixture.Identifier}: {fixture.Label}");
+            return result;
+        }
+        result.Add($"Testing fixture {fixture.Identifier}: {fixture.Label}");
+        var client = new HttpClient();
```

Comment:
I doubt it'd cause an issue but maybe worth disposing of instantiating this outside of `SmokeTester` and passing it in so that the same one is reused across all test runs.